src/main/java/lesson44/repository/BankingRepository.java

```diff
@@ -0,0 +1,111 @@
+package lesson44.repository;
+
+import lesson44.dao.TransferCardDTO;
+import lesson44.model.CardModel;
+import lesson44.model.ClientModel;
+import lesson44.pack.PostgresDriverManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class BankingRepository {
+    @Autowired
+    public PostgresDriverManager postgresDriverManager;
+
+    public ClientModel getClientById(int id) {
+        String sql = """
+                SELECT client.client_id,client.name,card.number,card.card_id,card.balance FROM client
+                JOIN card ON client.client_id = card.client_id
+                WHERE client.client_id = ?;
+                """;
```

Comment:
Why do you have these on the new line?

Reply:
It is more pleasant for the eye to see the SQL request

Reply:
But it's not pleasant to see a lot of quotes